src/test/librbd/fsx.cc

```diff
@@ -1183,7 +1185,22 @@ krbd_resize(struct rbd_ctx *ctx, uint64_t size)
 	if (ret < 0)
 		return ret;
 
-	return __librbd_resize(ctx, size);
+	ret = __librbd_resize(ctx, size);
+	if (ret < 0)
+		return ret;
+	do {
+		sleep(count++);
```

Comment:
@idryomov you might have missed to notice `count = 0` (i.e. count is set to 0, which makes it No op the first time).

Reply:
Looks like I indeed missed that.  But then I don't see a point in calling something knowing that it would be a no-op...

Reply:
I didn't had a limiter before hence I decided to use do{}while as I we have to call the get_size at-least once and validate the size. Anyway, since now you asked me to have a boundary to break that loop, we don't need a do{}while. Please check the latest changes. Thanks!